packages/@romejs/js-compiler/transforms/lint/disallowMultipleSpacesInRegularExpressionLiterals.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {AnyNode} from '@romejs/js-ast';
+import {Path} from '@romejs/js-compiler';
+
+const multipleSpacesPattern = /( {2,})(?: [+*{?]|[^+*{?]|$)/gu;
+
+export default {
+  name: 'disallowMultipleSpacesInRegularExpressionLiterals',
+  enter(path: Path): AnyNode {
+    const {context, node} = path;
+
+    if (
+      (node.type === 'NewExpression' &&
+        node.callee.type === 'ReferenceIdentifier' &&
+        node.callee.name === 'RegExp' &&
+        node.arguments.length > 0 &&
+        node.arguments[0].type === 'StringLiteral' &&
+        multipleSpacesPattern.test(node.arguments[0].value)) ||
+      (node.type === 'ExpressionStatement' &&
+        node.expression.type === 'RegExpLiteral' &&
+        multipleSpacesPattern.test(node.expression.pattern))
```

Comment:
This should be:

```suggestion
      (node.type === 'RegExpLiteral' &&
        multipleSpacesPattern.test(node.pattern))
```

It's overly specific because `ExpressionStatement` is only used as the wrapper statement for an expression. A regexp should never appear on a line by itself.

Reply:
I see and accept suggestion, also fix conflict as soon as possible.